pkg/apis/jenkins.io/v1/types_pipeline.go

```diff
@@ -94,13 +95,14 @@ type PipelineActivityStep struct {
 
 // CoreActivityStep is a base step included in Stages of a pipeline or other kinds of step
 type CoreActivityStep struct {
-	Name               string             `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
-	Description        string             `json:"description,omitempty" protobuf:"bytes,2,opt,name=description"`
-	Status             ActivityStatusType `json:"status,omitempty" protobuf:"bytes,3,opt,name=status"`
+	Name        string              `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
+	Description string              `json:"description,omitempty" protobuf:"bytes,2,opt,name=description"`
+	Status      ActivityStatusType  `json:"status,omitempty" protobuf:"bytes,3,opt,name=status"`
+	Message     ActivityMessageType `json:"message,omitempty" protobuf:"bytes,3,opt,name=message"`
```

Comment:
This will add message to the sub steps of the stage step (and promote and preview steps as well), I think that is ok for now. May be in the future, we can use this for something else ...